primitives/header-chain/src/lib.rs

```diff
@@ -225,6 +225,13 @@ pub trait ChainWithGrandpa: Chain {
 	/// the same name.
 	const WITH_CHAIN_GRANDPA_PALLET_NAME: &'static str;
 
+	/// Name of the runtime API method that is returning the GRANDPA justifications accepted
+	/// by the `submit_finality_proofs` extrinsic in the queried block.
+	///
+	/// Keep in mind that this method is normally provided by the other chain, which is
+	/// bridged with this chain.
+	const ACCEPTED_FINALITY_PROOFS_METHOD: &'static str;
```

Comment:
Do we really need this at primitives level? We don't need it in the runtime, only from relay code - maybe worth moving it to `relays/*`?

Reply:
Done.